### PR TITLE
bpo-1635741: Port statistics module to multiphase initialization

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue1635741

### CODE REVIEW

**Comment by corona10** [comment]
@shihai1991 Can you please take a look?

**Review by shihai1991 — APPROVED** [review]
I have checked this source file(there have no _Py_IDENTIFIER or global vars), so this PR LGTM, thanks.

**Comment by shihai1991** [comment]
let us ping experts @vstinner @encukou @ncoghlan

File: Misc/NEWS.d/next/Core and Builtins/2020-03-15-20-51-15.bpo-1635741.iH0JND.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Port statistics  module to multiphase initialization (:pep:`489`).
```
Comment (vstinner) [inline Misc/NEWS.d/next/Core and Builtins/2020-03-15-20-51-15.bpo-1635741.iH0JND.rst]:
```suggestion
Port _statistics module to multiphase initialization (:pep:`489`).
```

**Review by vstinner — COMMENTED** [review]
LGTM, but please fix the typo in the commit message and in the NEWS entry: it's the private _statistics module, not statistics :-)

**Pull request merged**